crmsh/sbd.py

```diff
@@ -47,6 +47,8 @@ def __init__(self, context):
         self._sbd_watchdog_timeout = 0
         self._is_s390 = "390" in os.uname().machine
         self._context = context
+        self._sbd_msgwait = 10
```

Comment:
Be aware correspondingly, if msgwait is defaulted on `sbd create`, sbd uses 30 for s390(x) otherwise 10, basically 2 * default_watchdog_timeout.

Reply:
@gao-yan I added this profile, and make sure this will be loaded when in s390:
```
s390:
  sbd.watchdog_timeout: 15
  sbd.msgwait: 30
```

Reply:
Like mentioned, in case of s390, if an user unset this setting from profile s390 or even deleted the profile s390, it'd be a wrong value to be used.

OTOH, frankly I don't see much value of explicitly duplicating the internal default values in the shipped profiles. We anyway could not rely on them to be as internal defaults, since they could be changed or unset by users.

We should make it somehow work properly in cases of real "default".

* Any settings are unset in profiles
* Any specific profiles are missing
* Probably even the whole profiles.yml is missing

 by defaulting

Reply:
There are prirority levels. The intuitive order would be,

specific profile if detected > "default" profile > crmsh defaults if any > internal defaults of sbd/corosync

These priority logic is simple and consisence enough in my view. Even, it can be evolve further. For example, if any users has demand, we can implement the logic further to allow users to manage the sperate "profile-01-xxxx.yml", "profile-02-xxxx.yml", if they want. These file will have the highest priority to override all others.